geonode/contrib/edit_data/templates/edit_data/wfs_delete_row.xml

```diff
@@ -0,0 +1,9 @@
+<wfs:Transaction service="WFS" version="1.0.0"
+  xmlns:ogc="http://www.opengis.net/ogc"
+  xmlns:wfs="http://www.opengis.net/wfs">
+  <wfs:Delete typeName="geonode:{{layer_name}}">
```

Comment:
the same thing for the typeName

Reply:
@hishamkaram Thanks for the comment. I have updated the code. It is not clear to me why the tests are failing. Could you please have a look as well?